lighthouse-core/test/config/config-test.js

```diff
@@ -64,6 +64,10 @@ describe('Config', () => {
         super();
         this.secret = secretVal;
       }
+
+      get name() {
+        return `MyGatherer${this.secret}`;
```

Comment:
```suggestion
        // Use unique artifact name per instance so gatherers aren't deduplicated.
        return `MyGatherer${this.secret}`;
```

probably also worth a comment. It looks like something that's important to the test (going to be spied on or whatever), as opposed to just keeping the artifact uniquely named :)